cnv/cnv_users_guide/cnv-installing-qemu-guest-agent.adoc

```diff
@@ -4,17 +4,30 @@ include::modules/cnv-document-attributes.adoc[]
 :context: cnv-installing-qemu-guest-agent
 toc::[]
 
+.Prerequisites
+
+* Verify that the guest agent is installed and running by entering the following command:
++
+----
+'systemctl status qemu-guest-agent'
```

Comment:
Commands need to start with $ or #, and unless it's required as part of the command syntax, it shouldn't be enclosed in single quotation marks.

Reply:
Done.